CMakeLists.txt

```diff
@@ -281,8 +281,22 @@ else()
         "-ldl -Wl,--whole-archive -Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/scripts/xquic.lds"
         ${SSL_LIB_PATH}
         "-Wl,--no-whole-archive"
-        -lpthread
-    )
+        -lpthread)
+
+    if(CMAKE_SYSTEM_NAME MATCHES "Windows")
+        target_link_libraries(xquic -lws2_32 -lgdi32 -lcrypt32)
```

Comment:
Are you having problems when compile，need to add -lws2_32 -lgdi32 -lcrypt32，Can you describe the problem you're having

Reply:
yes, I have many unresolved function symbols about the socket. and others.